module/skill/GetUp/src/GetUp.hpp

```diff
@@ -39,6 +39,7 @@ namespace module::skill {
     private:
         /// @brief Stores configuration values
         struct Config {
+            int delay_time; // milliseconds
```

Comment:
```suggestion
            /// @brief The time to delay before getting up, in milliseconds, to settle the sensors for accurate side detection 
            int delay_time = 0; // milliseconds
```
or something like that. good habits to get into, initialising variables and adding documentation directives (the `@brief`)